### PR TITLE
Added range circles to Nuke and Railgun support powers

### PR DESCRIPTION
**Pull request by OcelotWalrus**
Added a range circle to both Nuke and Railgun support powers, to show the damage range to the player.

### CODE REVIEW

File: mods/hv/rules/buildings.yaml [hunk 1]
```diff
@@ -1588,6 +1590,7 @@ UPLINK:
 		SelectTargetTextNotification: notification-select-target
 		IncomingSpeechNotification: OrbitalRailgunFired
 		AirburstAltitude: 40c0
+		TargetCircleRange: 5c0
```
Comment (Mailaender) [inline mods/hv/rules/buildings.yaml:1593]:
```suggestion
		TargetCircleRange: 3c0
```

File: mods/hv/rules/buildings.yaml [hunk 2]
```diff
@@ -1506,6 +1506,8 @@ SILO:
 		SelectTargetTextNotification: notification-select-target
 		IncomingSpeechNotification: NukeIncoming
 		OrderName: Nukem
+		CircleRanges: 25c0
```
Comment (Mailaender) [inline mods/hv/rules/buildings.yaml:1509]: This is more than the entire screen.
grafik

```suggestion
		CircleRanges: 6c0
```
maybe

File: mods/hv/rules/buildings.yaml (hunk 1 above)
Comment (OcelotWalrus) [reply mods/hv/rules/buildings.yaml:1593]: I actually tested by comparing the range circle and the red damage circle with the combat geometry overlay and it fits correctly.

File: mods/hv/rules/buildings.yaml (hunk 2 above)
Comment (OcelotWalrus) [reply mods/hv/rules/buildings.yaml:1509]: I know it's huge but as with the uplink, it fits the damage zone. It even is a bit smaller.

Comment (OcelotWalrus) [reply mods/hv/rules/buildings.yaml:1509]: On my screen it doesn't.

Comment (Mailaender) [reply mods/hv/rules/buildings.yaml:1509]: You are including the fall off.

**Pull request merged**